tools/importPlanGoals.test.js

```diff
@@ -7,6 +8,39 @@ describe('Import TTA plan goals', () => {
   afterAll(() => {
     db.sequelize.close();
   });
+  describe('Update grants and grantees', () => {
+    beforeAll(async () => {
+      await Grant.destroy({ where: {} });
+      await Grantee.destroy({ where: {} });
+    });
+    afterEach(async () => {
+      await Grant.destroy({ where: {} });
+      await Grantee.destroy({ where: {} });
+    });
+    it('should import or update grantees', async () => {
+
+      const granteesBefore = await Grantee.findAll();
+      expect(granteesBefore.length).toBe(0);
+      await updateGrantsGrantees();
+
+      const grantee = await Grantee.findOne({ where: { id: 1335 } });
+      expect(grantee).toBeDefined();
+      expect(grantee.name).toBe('Agency 1, Inc.');
+    });
+
+    it('should import or update grants', async () => {
+      const grantsBefore = await Grantee.findAll();
```

Comment:
I think you meant to check the count of `Grant` before the test?

```suggestion
      const grantsBefore = await Grant.findAll();
```

Reply:
Good catch!